overlord/snapstate/backend/mountunit.go

```diff
@@ -44,3 +46,10 @@ func removeMountUnit(mountDir string, meter progress.Meter) error {
 	sysd := systemd.New(systemd.SystemMode, meter)
 	return sysd.RemoveMountUnitFile(mountDir)
 }
+
+func (b Backend) RemoveSnapMountUnits(s snap.PlaceInfo, meter progress.Meter) error {
+	if err := wrappers.RemoveMountUnitFiles(s, "", meter); err != nil {
+		logger.Noticef("Cannot remove mount units for %q: %v", s.InstanceName(), err)
```

Comment:
I'm not sure what the intended behaviour is here regarding failures. Can you explain to me why is this logged? `wrappers.RemoveMountUnitFiles` returns an error immediately on failure to remove a mount so, if one removal fails, RemoveMountUnitFiles won't try to remove the others but the caller of this method also can't react since it never gets an error.

Reply:
You are right, this does not make much sense. It's a leftover from when I moved this code away from `link.go`, where this actually made sense. I'll fix it. :-)

Reply:
Thanks :slightly_smiling_face: I think there's some unit tests failing, probably related to this change